test_backward_compatibility.py

```
@@ -0,0 +1,80 @@
+
+print("Testing MCP backward compatibility...")
+print("=" * 50)
+
+# Test 1: Existing SSE URL usage (should work unchanged)
+print("\n1. Testing existing SSE URL usage:")
+try:
+    mcp_sse = MCP("http://localhost:8080/sse")
+    print("✓ SSE URL initialization successful")
+    print(f"  - is_http: {getattr(mcp_sse, 'is_http', 'Not set')}")
+    print(f"  - is_sse: {getattr(mcp_sse, 'is_sse', 'Not set')}")
+except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)